src/test/ui/consts/const-extern-fn/abi-mismatch.rs

```diff
@@ -0,0 +1,16 @@
+// Checks that we report ABI mismatches for "const extern fn"
+// compile-flags: -Z unleash-the-miri-inside-of-you
```

Comment:
We usually have all miri-unleash tests in the corresponding folder, so I think we should do the same here... Cc @oli-obk